src/library/searchqueryparser.h

```diff
@@ -36,6 +43,7 @@ class SearchQueryParser {
 
     QRegExp m_fuzzyMatcher;
     QRegExp m_textFilterMatcher;
+    QRegExp m_crateFilterMatcher;
```

Comment:
You forgot to remove this now that m_textFilterMatcher is used for the crate filter.